osu.Framework/Text/TextBuilder.cs

```diff
@@ -62,24 +70,26 @@ public float LineBaseHeight
         /// </summary>
         /// <param name="store">The store from which glyphs are to be retrieved from.</param>
         /// <param name="font">The font to use for glyph lookups from <paramref name="store"/>.</param>
-        /// <param name="useFontSizeAsHeight">True to use the provided <see cref="font"/> size as the height for each line. False if the height of each individual glyph should be used.</param>
+        /// <param name="useFullGlyphHeight">True to use <see cref="TextBuilderGlyph.Size"/> (full glyph size) as the height for each line. False if the height of each individual glyph should be used.</param>
```

Comment:
Not sure what the renames of this parameter are about but I don't understand what it does either from the old code or the new code.

> **(full glyph size)** as the height for each line. False if the height of **each individual glyph** should be used.

What is the difference between those "glyph sizes"? I don't get it.

Reply:
"Full glyph size" refers to the size provided by the font usage (+ CSS scaling), while "height of each individual glyph" refers to the height of each glyph texture without any padding applied to it (using [YOffset + Height](https://www.angelcode.com/products/bmfont/doc/render_text.html)).

Would changing to:
```suggestion
        /// <param name="useFullGlyphHeight">True to use <see cref="TextBuilderGlyph.Size"/> (full glyph size) as the height for each line. False if the height of each individual glyph (<see cref="TextBuilderGlyph.YOffset"/> + <see cref="TextBuilderGlyph.Height"/>) should be used.</param>
```
explain it better?

Could also add another note that this is mainly used to allow centering text on a container (although it doesn't exactly help right now, since `YOffset` is still a problem).

Reply:
I don't know. Feels like this flag would be easier to explain inverted (as a "trim glyphs" sort of setting). But I wouldn't do that in this diff. Maybe fine then, I give in.